### PR TITLE
fix: sync cloud icon in search results

### PR DESCRIPTION
**Pull request by 5h1rU**
in result list sync the cloud icon stack context is set with a high z-index value.

**before**
Screen Shot 2021-09-07 at 9 32 21 AM

**after**
Screen Shot 2021-09-07 at 9 31 51 AM

### CODE REVIEW

File: client/shared/src/components/LastSyncedIcon.module.scss [hunk 1]
```diff
@@ -3,6 +3,6 @@
     right: 0.5rem;
     top: 0.5rem;
     height: 1rem;
-    z-index: 1000;
```
Comment (camdencheek) [inline client/shared/src/components/LastSyncedIcon.module.scss:6]: Overriding `z-index` with dev tools on `sourcegraph.com` and replacing it with `isolate` seems to make the icon disappear entirely behind the text content of the result.

**Review by camdencheek — CHANGES_REQUESTED** [review]

File: client/shared/src/components/LastSyncedIcon.module.scss (hunk 1 above)
Comment (camdencheek) [reply client/shared/src/components/LastSyncedIcon.module.scss:6]:
```css
element {
    isolation: isolate;
    z-index: unset;
}
```
Screen Shot 2021-09-07 at 08 45 26

Comment (5h1rU) [reply client/shared/src/components/LastSyncedIcon.module.scss:6]: Thanks, adding a z-index to fix this scenario.

**Review by camdencheek — APPROVED** [review]

**Review by valerybugakov — APPROVED** [review]

File: client/shared/src/components/LastSyncedIcon.module.scss [hunk 2]
```diff
@@ -3,6 +3,7 @@
...
+    isolation: isolate;
```
Comment (limitedmage) [inline client/shared/src/components/LastSyncedIcon.module.scss:6]: Thinking the `isolation: isolate` should be in the parent container, not the element itself?

**Pull request merged**